### DISCUSSION TITLE
connection setting in dbeaver for mysql

### DISCUSSION THREAD

**Discussion by neethumolvinod** [dbeaver/dbeaver#18935] [Q&A]
### Description

Hi, could you please help regarding how to set the connection timeout and query timeout setting in dbeaver for mysql?

Thanks!

### DBeaver Version

Community Edition 22.1.2

### Operating System

macOS Monterey

### Database and driver

MySQL 8

**Comment by LonwoLonwo** [comment]
Hello @neethumolvinod 

You can find and change these properties in the Driver Properties tab during the new connection creation.

![2023-02-01 19_18_56-Window](https://user-images.githubusercontent.com/45152336/216101040-7a85c62c-c7e9-40ac-b6ec-714a70f39cd9.png)

![2023-02-01 19_19_19-Window](https://user-images.githubusercontent.com/45152336/216101044-61a3d550-6eac-49b2-9e8e-fc1f46eb4032.png)

Or after the connection creation, you can change these settings in the connection settings:

![2023-02-01 19_20_11-Window](https://user-images.githubusercontent.com/45152336/216101338-34cf2d13-88d4-434a-965c-678a8b0e315b.png)

**Comment by neethumolvinod** [comment]
Hi, thanks for the reply. these are connection timeouts i believe, is there a place where we can set the query timeout too, like the max execution time for a query?

**Comment by LonwoLonwo** [comment]
Probably this one can help

![2023-02-01 20_25_32-Window](https://user-images.githubusercontent.com/45152336/216117440-6223a67f-c29d-4f5b-9090-e2a9d371be92.png)
